CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 [1]: https://pypi.org/project/demisto-py/#history
 ## 2.0.9
-Improved error message when missing authentication parameters.
+* Improved error message when missing authentication parameters.
```

Comment:
Add the change log entries to a new version: `2.0.10`. 2.0.9 was released alread.

Reply:
done.